### PR TITLE
Minor README & languages + parser 

### PR DESCRIPTION
**Pull request by MarcosMoreu**

### CODE REVIEW

**Comment by MarcosMoreu** [comment]
@tcouch just need to review the last commit. I should be okay now

File: src/languages.js [hunk 1]
```diff
@@ -22,7 +22,7 @@ i18next.init({
 				instructions: `<span class="title">Create WhatsApp Maps with Kapta </span><hr> 1 - Share locations in a WhatsApp group<br> 2 - Export chat to Kapta<br> 3 - Share your WhatsApp Map`,
 				watchtutorial: "Watch tutorial",
 				tutorialUrl:
-					"https://www.youtube.com/embed/PDOVl8smi2Q?si=CiIkw6_rm1TzmZgF",
```
Comment (tcouch) [inline src/languages.js:25]: The old tutorialURL is still present in the other languages. I'd suggest just removing that entry for languages which don't have a translated video. In that case it'd fall back to english.

File: src/import_whatsapp.js [hunk 2]
```diff
@@ -164,7 +164,7 @@ function processText(text) {
 
 	// Regex to match google maps location and capture lat (group 1) and long (group 2)
 	const locationRegex =
-		/location: https:\/\/maps\.google\.com\/\?q=(-?\d+\.\d+),(-?\d+\.\d+)/g;
+		/: https:\/\/maps\.google\.com\/\?q=(-?\d+\.\d+),(-?\d+\.\d+)/g; //Without 'location' to be universal - the word in the export file changes based on WA language
```
Comment (tcouch) [inline src/import_whatsapp.js:167]: Looks good, tested on my machine.

**Review by tcouch — CHANGES_REQUESTED** [review]

File: src/languages.js (hunk 1 above)
Comment (MarcosMoreu) [reply src/languages.js:25]: yeah, but better to have an english one than none. I'm awaiting the translations for am, pt, yo. I should get them soon

Comment (MarcosMoreu) [reply src/languages.js:25]: oh okay, I thought I updated the url. So now is the same url as English @tcouch

File: README.md [hunk 3]
```diff
@@ -2,14 +2,13 @@
 
 ## What? ###
 
-Kapta Mobile is a Progressive Web App enabling users to generate maps from WhatsApp chats.
+Kapta Mobile is a Progressive Web App to create WhatsApp Maps. 3 steps:
```
Comment (acholyn) [inline README.md:5]: I think the previous wording implying that the maps are generated from the whatsapp data makes it more obvious what we're doing
If you want to keep it short, maybe "create maps from Whatsapp in 3 steps:..."

File: README.md [hunk 4]
```diff
@@ -63,15 +62,6 @@ Kapta requires a configuration file to be created in the src directory. The file
   }
 }
 ```
-
-## Usage
```
Comment (acholyn) [inline README.md:67]: why remove this section?

File: src/languages.js [hunk 5]
```diff
@@ -95,7 +95,7 @@ i18next.init({
 				instructions: `<span class="title">Créer des WhatsApp Maps avec Kapta</span><hr>1 - Partagez des localisations dans un groupe WhatsApp<br>2 - Exportez la discussion dans Kapta<br>3 - Partagez votre WhatsApp Map`,
 				watchtutorial: "Regarder le tutoriel",
 				tutorialUrl:
-					"https://www.youtube.com/embed/Dd4dFC-J_Bo?si=CvzkDasZX-ocN-K2",
+					"https://youtube.com/embed/3KrsKgFHYMs?si=liiXRfVzN4N4YMXk",
```
Comment (acholyn) [inline src/languages.js:98]:
```suggestion
					"https://www.youtube.com/embed/3KrsKgFHYMs?si=liiXRfVzN4N4YMXk",
```

File: src/languages.js [hunk 6]
```diff
@@ -22,7 +22,7 @@ i18next.init({
 				instructions: `<span class="title">Create WhatsApp Maps with Kapta </span><hr> 1 - Share locations in a WhatsApp group<br> 2 - Export chat to Kapta<br> 3 - Share your WhatsApp Map`,
 				watchtutorial: "Watch tutorial",
 				tutorialUrl:
-					"https://www.youtube.com/embed/PDOVl8smi2Q?si=CiIkw6_rm1TzmZgF",
+					"https://youtube.com/embed/vaPHy8S-OpA?si=w6ou_ekAzkZbHJ3i",
```
Comment (acholyn) [inline src/languages.js:25]:
```suggestion
					"https://www.youtube.com/embed/vaPHy8S-OpA?si=w6ou_ekAzkZbHJ3i",
```

File: README.md (hunk 4 above)
Comment (MarcosMoreu) [reply README.md:67]: It is redundant with the "What?" section, isn't?

File: src/languages.js (hunk 5 above)
Comment (MarcosMoreu) [reply src/languages.js]: why www needed?

Comment (acholyn) [reply src/languages.js]: consistency

**Review by tcouch — APPROVED** [review]

**Pull request merged**